### ISSUE TITLE
regtest: for id'ing a user

### ISSUE THREAD

**Issue by maxtaco** [keybase/go#195]
After the [canonical users patch]() is applied, you should be able to track `t_alice`, `t_bob` or `t_charlie`.

**Comment by oconnor663** [comment]
there's now also t_doug (PGP key only, no sigchain) and t_ellie (no keys at all)

**Comment by oconnor663** [comment]
Pushed branch `jack_issue195`. Right now I'm only looking at the success of the operation, and I should probably look a little more deeply to confirm which proofs get validated. @maxtaco I'll ask you about that tomorrow.

**Comment by maxtaco** [comment]
Yeah, this is a good start.  You probably want to have a pointer method to your FakeUI so you can collect data as a result of the callbacks.
